python_modules/libraries/dagster-airbyte/dagster_airbyte/translator.py

```diff
@@ -186,5 +190,5 @@ def get_asset_spec(self, props: AirbyteConnectionTableProps) -> AssetSpec:
         return AssetSpec(
             key=AssetKey(props.table_name),
             metadata=metadata,
-            kinds={"airbyte"},
+            kinds={"airbyte", props.destination_type},
```

Comment:
Do we need to confirm that we have kind images for all destination types returned by the Airbyte API?

Reply:
There are more than [550 connectors](https://airbyte.com/connectors) in Airbyte, so I don't think we have all of them. That said, we already do this for Fivetran, which has [660 connectors](https://www.fivetran.com/connectors). We have the most important ones already.

Reply:
Ok that makes sense--just checking! Thanks.